code/edgeconsent/src/main/java/com/adobe/marketing/mobile/edge/consent/Consent.java

```diff
@@ -72,7 +76,7 @@ public static void registerExtension() {
 	 *
 	 * @param consents A {@link Map} of consents to be merged with the existing consents
 	 */
-	public static void update(final Map<String, Object> consents) {
+	public static void update(@NonNull final Map<String, Object> consents) {
```

Comment:
Add @NonNull to return type for extensionVersion and getFriendlyName and getVersion in Extension class.

Refer: https://github.com/adobe/aepsdk-edgeidentity-android/pull/83